src/main/kotlin/com/terraformation/backend/accelerator/db/CohortStore.kt

```diff
@@ -36,22 +35,19 @@ class CohortStore(
   fun fetchOneById(
```

Comment:
Changes in this file, and the removal of CohortModuleDepth and CohortModuleModel are to do with removing functionality in this store to query for modules. 

The functionalities are moved to CohortModuleStore, and using ModuleModel instead.